src/tests/infos_docs.edn

```diff
@@ -0,0 +1,142 @@
+; run with ./build/cbl.exe ./src/tests/infos_docs.edn
+(defnode main)
```

Comment:
I would use clojure like formatting 😄

Reply:
What do you mean?

Reply:
https://github.com/bbatsov/clojure-style-guide

Reply:
AFAIK according to the guide (which I don't like because some rules contradict each other), it is already in the right form (single argument is fine on the same line).

Reply:
it is, I think just the way to close the block is not, that's it.

Reply:
but we can keep this way for now anyway